src/test/java/com/pontusvision/gdpr/pbr/PVPbrTest.java

```diff
@@ -44,37 +42,38 @@ public void test00001FontesDeDados() throws InterruptedException {
-                        ".as('fontes-de-dados').values('Object_Data_Source_Description').next().toString()").get().toString();
-        assertEquals("EQUIPAMENTOS DE PROTEÇÃO INDIVIDUAL", getFontesEquipamentosDescription, "Description of EQUIPAMENTOS");
-
-        String getFontes4System =
-                App.executor.eval("App.g.V().has('Object_Data_Source_Name', eq('SHAREPOINT/PBR/FONTES-DE-DADOS')).as('root')" +
-                        ".out('Has_Ingestion_Event').has('Event_Group_Ingestion_Type', eq('sharepoint/pbr/fontes-de-dados')).as('group-ingestion')" +
-                        ".out('Has_Ingestion_Event').has('Event_Ingestion_Type', eq('sharepoint/pbr/fontes-de-dados')).as('event-ingestion')" +
-                        ".out('Has_Ingestion_Event').has('Object_Data_Source_Form_Id', eq('four')).as('fontes-de-dados')" +
-                        ".out('Has_Module').has('Object_Module_Name', eq('FIXA')).as('tabela')" +
-                        ".out('Has_Subsystem').has('Object_Subsystem_Name', eq('ONPREM')).as('onPrem')" +
-                        ".out('Has_System').as('system')" +
-                        ".values('Object_System_Name').next().toString()").get().toString();
-        assertEquals("POLARIS", getFontes4System, "System of fontes-de-dados 4");
+        App.executor.eval("App.g.V().has('Event_Ingestion_Type', eq('sharepoint/pbr/fontes-de-dados'))" +
+          ".as('event-ingestions').out('Has_Ingestion_Event').has('Object_Data_Source_Form_Id', eq('two'))" +
+          ".as('fontes-de-dados-two').out('Has_Sensitive_Data').dedup().count().next().toString()").get().toString();
+      assertEquals("5", countDadosFontes2,
+        "Apelido, Data da Aposentadoria, Situação no CIPA, Classificação do visto, PIS/PASEP/NIT");
+
+      String getFontesEquipamentosDescription =
+        App.executor.eval("App.g.V().has('Event_Ingestion_Type', eq('sharepoint/pbr/fontes-de-dados'))" +
+          ".as('event-ingestions').out('Has_Ingestion_Event').has('Object_Data_Source_Name', eq('EQUIPAMENTOS'))" +
+          ".as('fontes-de-dados').values('Object_Data_Source_Description').next().toString()").get().toString();
+      assertEquals("EQUIPAMENTOS DE PROTEÇÃO INDIVIDUAL", getFontesEquipamentosDescription, "Description of EQUIPAMENTOS");
+
+      String getFontes4System =
+        App.executor.eval("App.g.V().has('Object_Data_Source_Name', eq('SHAREPOINT/PBR/FONTES-DE-DADOS')).as('root')" +
+          ".out('Has_Ingestion_Event').has('Event_Group_Ingestion_Type', eq('sharepoint/pbr/fontes-de-dados')).as('group-ingestion')" +
+          ".out('Has_Ingestion_Event').has('Event_Ingestion_Type', eq('sharepoint/pbr/fontes-de-dados')).as('event-ingestion')" +
+          ".out('Has_Ingestion_Event').has('Object_Data_Source_Form_Id', eq('four')).as('fontes-de-dados')" +
+          ".out('Has_Module').has('Object_Module_Name', eq('fixa')).as('tabela')" +
```

Comment:
Please check this; I thought we had uppercase here before in the Object_Module_Name